index.bs

```diff
@@ -5741,7 +5741,7 @@ which just allows the additional value <emu-val>null</emu-val>
 to be a member of its set of values.  [=Nullable types=]
 are represented in IDL by placing a <span class="char">U+003F QUESTION MARK ("?")</span>
 character after an existing type.  The inner type must not
-be {{any}},
+be {{any}}, a [=Promise type=],
 another nullable type, or a [=union type=]
```

Comment:
It seems OK to me. Maybe a bulleted list would make it clearer. "The inner type must not be any of the following:

- any
- a Promise type
- another nullable type
- a union type that itself has includes a nullable type [sic; maybe fix this while here] or has a dictionary or record type as one of its flattened member types"